src/main/java/uk/gov/hmcts/probate/service/filebuilder/HmrcFileService.java

```diff
@@ -103,6 +103,14 @@ private int prepareCaseData(ImmutableList.Builder<String> fileData, Long id, Cas
             fileData.add(getPoundValue(data.getIhtNetValue()));
             fileData.add(DataExtractGrantType.valueOf(data.getCaseType()).getCaseTypeMapped());
             fileData.add(FINAL_GRANT);
+            fileData.add(getPoundValue(data.getIhtEstateGrossValue()));
+            fileData.add(getPoundValue(data.getIhtEstateNetValue()));
+            fileData.add(getPoundValue(data.getIhtEstateNetQualifyingValue()));
+            fileData.add(Optional.ofNullable(data.getIhtUnusedAllowanceClaimed()).orElse(""));
```

Comment:
maybe its worth putting this Optional.. instantiation into its own method, so you can reuse it?